docs/spec-files/encryption.md

```diff
@@ -101,7 +101,7 @@ Assuming she also has Bob's DID document which happens to include the following
 }
 ```
 
-There should be only 1 entry in the recipients of the envelope, representing Bob. The corresponding `kid` header for this recipient MUST have `did:example:jklmnopqrstuvwxyz1#key-1` as value. The packing process MUST extract the public key bytes found in `publicKeyBase58` of Bob's DID Doc `KeyAgreement[0]` to execute the ECDH-1PU key derivation for content key wrapping.
+The recipients of the envelope MUST include all the KeyAgreement entries, representing Bob. The corresponding `kid` header for this recipient MUST have a DID URL pointing to a verification method in the did document. This verification method MUST be associated with the `KeyAgreement` verification relationship and the public key bytes MUST be retrieved from the DID Document to execute the ECDH-1PU key derivation for content key wrapping.
```

Comment:
```suggestion
The recipients of the envelope MUST include all the KeyAgreement entries, representing Bob. The corresponding `kid` header for this recipient MUST have a DID URL pointing to a verification method in the DID Document. This verification method MUST be associated with the `KeyAgreement` verification relationship and the Verification Material MUST be retrieved from the DID Document to execute the ECDH-1PU key derivation for content key wrapping.
```

Reply:
I don't agree with "MUST" here. I think "SHOULD" is more appropriate. We can discuss on a call.

Reply:
Sure, I'll replace MIUST with SHOILD this this case and add `Unless previously coordinated, the recipients of the ...`

As suggested by @awoie, i'll push a separate PR for this change and revert it here.

Reply:
pr created: https://github.com/decentralized-identity/didcomm-messaging/pull/270